x/rollapp/types/params.go

```diff
@@ -12,10 +12,10 @@ import (
 var _ paramtypes.ParamSet = (*Params)(nil)
 
 var (
-	// KeyRollappsEnabled is store's key for RollappsEnabled Params
-	KeyRollappsEnabled = []byte("RollappsEnabled")
-	// KeyDeployerWhitelist is store's key for DeployerWhitelist Params
-	KeyDeployerWhitelist = []byte("DeployerWhitelist")
+	// KeyRegistrationFee is store's key for RegistrationFee Params
+	KeyRegistrationFee = []byte("RegistrationFee")
+	// DefaultRegistrationFee is the default registration fee
+	DefaultRegistrationFee, _ = sdk.ParseCoinNormalized("1000000000000000000adym")
```

Comment:
this syntax is better IMO
`sdk.NewCoin(appparams.BaseDenom, DYM.Mul(sdk.NewInt(1000))) // 1000DYM`